compiler/il/ILOpCodeProperties.hpp

```diff
@@ -264,6 +264,118 @@
    /* .ifCompareOpCode      = */ TR::BadILOp,
    },
 
+   {
+   /* .opcode               = */ TR::irdbar,
+   /* .name                 = */ "irdbar",
+   /* .properties1          = */ ILProp1::LoadVar | ILProp1::HasSymbolRef,
+   /* .properties2          = */ ILProp2::ValueNumberShare | ILProp2::SupportedForPRE| ILProp2::MayUseSystemStack | ILProp2::ReadBarrierLoad,
```

Comment:
Does PRE really handle these?

Reply:
PRE hasn't been updated to include the support yet. Another PR will address optimizer changes to support this opcode but as a property for the opcode all the optimizations applied to loads should be able to be applied to rdbars.